src/components/FamilyManifestModal/index.js

```diff
@@ -1 +1,6 @@
 export { default } from './FamilyManifestModal';
+export {
+  familyMemberAndParticipantDataQueryBody,
+  dataTypeDataQueryBody,
+  generateFamilyManifestModalProps,
+} from './FamilyManifestModalData';
```

Comment:
a decent name for `FamilyManifestModalData.js` might just be `queries.js` since it's already namespaced in the folder, and the capital .js file typically implies a component